frcxx_nft/src/types.rs

```diff
@@ -55,7 +56,57 @@ pub trait FRCXXXNFT {
     fn is_approved_for_all(&self, params: IsApprovedForAllParams) -> bool;
 }
 
-#[derive(Serialize_tuple, Deserialize_tuple, Debug)]
+/// Return value after a successful mint
+/// The mint method is not standardised, so this is merely a useful library-level type, and
+/// recommendation for token implementations
+#[derive(Serialize_tuple, Deserialize_tuple, Clone, Debug)]
+pub struct MintReturn {
+    /// The new balance of the owner address
+    pub balance: u64,
+    /// The new total supply
+    pub supply: u64,
+    /// List of the tokens that were minted successfully (some may have been burned during hook execution)
+    pub token_ids: Vec<TokenID>,
+    /// (Optional) data returned from the receiver hook
+    pub recipient_data: RawBytes,
+}
+
+impl Cbor for MintReturn {}
```

Comment:
See https://github.com/filecoin-project/builtin-actors/issues/700, and please remove this from everywhere. Or lmk if you actually use it for something.

Reply:
https://github.com/helix-onchain/filecoin/pull/156